### ISSUE TITLE
[Issue]: Library content has disappeared, scanning fails

### ISSUE THREAD

**Issue by jeames00** [jellyfin/jellyfin#8195]
### Please describe your bug

Content within libraries has spontaneously disappeared. Furthermore, attempts to scan libraries end abruptly, with the following error in the log:

> Emby.Server.Implementations.Library.LibraryManager: Error running post-scan task
> System.ArgumentException: Not enough valid pictures provided to create a splashscreen!

Initially on version 10.7.7, everything was working alright. At some point I realised that metadata wasn't being downloaded for newly added content. I diagnosed this to be a DNS issue and so added a DNS server IP to the docker-compose config. Metadata for recently added media then had to be re-scanned. After manually triggering re-scans for several TV episodes, all content in both libraries disappeared. I updated to 10.8.1 but the issue persists. I have encountered this issue in the past and failed to resolve, the only workaround then was to trash the metadata/config directories and start again.

The "splash screen" option in the settings is not enabled.

### Jellyfin Version

Other

### if other:

10.8.1

### Environment

```markdown
- OS: TrueNAS > Alpine Linux VM > Debian docker image
- Virtualization: Docker on an Alpine Linux VM
- Clients: Browser
- Browser: Brave 1.41.100
- FFmpeg Version: 5.0.1
- Playback Method: Direct Play
- Hardware Acceleration: None
- Plugins: AudioDB (10.8.1.0), MusicBrainz (10.8.1.0), OMBd (10.8.1.0), Studio Images (10.8.1.0), TMDb (10.8.1.0)
- Reverse Proxy: None
- Base URL: None
- Networking: Host
- Storage: ZFS dataset shared via NFS
```

### Jellyfin logs

```shell
[2022-08-01 16:20:25.773 +10:00] [INF] [17] Emby.Server.Implementations.ScheduledTasks.TaskManager: Executing "Scan Media Library"
[2022-08-01 16:20:25.777 +10:00] [INF] [17] Emby.Server.Implementations.IO.LibraryMonitor: Stopping directory watching for path "/data/movies"
[2022-08-01 16:20:25.778 +10:00] [INF] [17] Emby.Server.Implementations.IO.LibraryMonitor: Stopping directory watching for path "/data/tv"
[2022-08-01 16:20:25.783 +10:00] [INF] [17] Emby.Server.Implementations.Library.LibraryManager: Validating media library
[2022-08-01 16:20:25.844 +10:00] [ERR] [17] Emby.Server.Implementations.Library.LibraryManager: Error running post-scan task
System.ArgumentException: Not enough valid pictures provided to create a splashscreen!
   at Jellyfin.Drawing.Skia.SplashscreenBuilder.GenerateCollage(IReadOnlyList`1 posters, IReadOnlyList`1 backdrops)
   at Jellyfin.Drawing.Skia.SplashscreenBuilder.GenerateSplash(IReadOnlyList`1 posters, IReadOnlyList`1 backdrops, String outputPath)
   at Jellyfin.Drawing.Skia.SkiaEncoder.CreateSplashscreen(IReadOnlyList`1 posters, IReadOnlyList`1 backdrops)
   at Emby.Server.Implementations.Library.SplashscreenPostScanTask.Run(IProgress`1 progress, CancellationToken cancellationToken)
   at Emby.Server.Implementations.Library.LibraryManager.RunPostScanTasks(IProgress`1 progress, CancellationToken cancellationToken)
[2022-08-01 16:20:25.864 +10:00] [INF] [17] Emby.Server.Implementations.ScheduledTasks.TaskManager: "Scan Media Library" Completed after 0 minute(s) and 0 seconds
[2022-08-01 16:20:25.867 +10:00] [INF] [17] Emby.Server.Implementations.ScheduledTasks.TaskManager: ExecuteQueuedTasks
[2022-08-01 16:20:25.901 +10:00] [INF] [19] Emby.Server.Implementations.IO.LibraryMonitor: Watching directory "/data/tv"
[2022-08-01 16:20:25.901 +10:00] [INF] [28] Emby.Server.Implementations.IO.LibraryMonitor: Watching directory "/data/movies"
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by hyperq** [comment]
same question

**Comment by VincentHui** [comment]
I had a clean install and had the same issue, as a new comer to jellyfish pretty sad. I've seen a couple of mentions of this issue but no clear way of fixing. I'm going to reinstall and then disable splash screen and see what happens

**Comment by capric98** [comment]
same issue, try to disable splash screen from dashboard -> general, doesn't work

**Comment by capric98** [comment]
https://github.com/jellyfin/jellyfin/blob/f863ca1f2d00839fd78a7655c759e25e9815483f/Jellyfin.Drawing.Skia/SplashscreenBuilder.cs#L88-L91

not familiar with csharp but i'm thinking if the thumb image is empty, the function will throw an exception rather than return an empty image?

**Comment by capric98** [comment]
> Post scan tasks (the SplashscreenPostScanTask) don't stop the library scan, so there is something else happening in your Jellyfin instance.
> 
> Does the scheduled tasks page show the "Scan Media Library" task as failed?

no, for me the scan progress bar just blink and disappear, no failure notification, but the library will not be updated, from F12 it seems like the scan task is "completed", later I could try debug mode and paste some log here...

**Comment by capric98** [comment]
> Post scan tasks (the SplashscreenPostScanTask) don't stop the library scan, so there is something else happening in your Jellyfin instance.
> 
> Does the scheduled tasks page show the "Scan Media Library" task as failed?

no, for me the scan progress bar just blink and disappear, no failure notification, but the library will not be updated, from F12 it seems like the scan task is "completed", later I could try debug mode and paste some log here...

```
[2022-11-22 15:57:10.870 +00:00] [INF] [19] Emby.Server.Implementations.ScheduledTasks.TaskManager: Executing "Scan Media Library"
[2022-11-22 15:57:10.878 +00:00] [INF] [19] Emby.Server.Implementations.IO.LibraryMonitor: Stopping directory watching for path "/media/samples"
[2022-11-22 15:57:10.881 +00:00] [INF] [19] Emby.Server.Implementations.Library.LibraryManager: Validating media library
[2022-11-22 15:57:10.894 +00:00] [DBG] [19] MediaBrowser.Providers.Folders.FolderMetadataService: Refreshing "Folder" "/config/root" because item.RequiresRefresh() returned true
[2022-11-22 15:57:10.912 +00:00] [DBG] [22] Jellyfin.Api.Auth.CustomAuthenticationHandler: AuthenticationScheme: "CustomAuthentication" was successfully authenticated.
[2022-11-22 15:57:10.912 +00:00] [DBG] [22] Jellyfin.Api.Auth.CustomAuthenticationHandler: AuthenticationScheme: "CustomAuthentication" was successfully authenticated.
[2022-11-22 15:57:10.927 +00:00] [DBG] [19] MediaBrowser.Providers.Folders.FolderMetadataService: Running "FolderImageProvider" for "/config/root"
[2022-11-22 15:57:11.136 +00:00] [DBG] [20] MediaBrowser.Providers.Folders.FolderMetadataService: "FolderImageProvider" reports change to "/config/root/default"
[2022-11-22 15:57:11.137 +00:00] [DBG] [20] MediaBrowser.Providers.Folders.FolderMetadataService: Running "FolderImageProvider" for "/config/root/default"
[2022-11-22 15:57:11.147 +00:00] [DBG] [13] MediaBrowser.Providers.Folders.CollectionFolderMetadataService: "CollectionFolderImageProvider" reports change to "/config/root/default/DIR"
[2022-11-22 15:57:11.148 +00:00] [DBG] [13] MediaBrowser.Providers.Folders.CollectionFolderMetadataService: Running "CollectionFolderImageProvider" for "/config/root/default/DIR"
[2022-11-22 15:57:11.151 +00:00] [DBG] [13] MediaBrowser.Providers.Folders.CollectionFolderMetadataService: "CollectionFolderImageProvider" reports change to "/config/root/default/DIR"
[2022-11-22 15:57:11.151 +00:00] [DBG] [13] MediaBrowser.Providers.Folders.CollectionFolderMetadataService: Running "CollectionFolderImageProvider" for "/config/root/default/DIR"
[2022-11-22 15:57:11.152 +00:00] [DBG] [13] MediaBrowser.Providers.Manager.ProviderManager: OnRefreshStart "f27caa37e5142225cceded48f6553502"
[2022-11-22 15:57:11.156 +00:00] [DBG] [13] MediaBrowser.Providers.Manager.ProviderManager: OnRefreshProgress "f27caa37e5142225cceded48f6553502" 5
[2022-11-22 15:57:11.159 +00:00] [DBG] [13] MediaBrowser.Providers.Manager.ProviderManager: OnRefreshProgress "f27caa37e5142225cceded48f6553502" 10
[2022-11-22 15:57:11.159 +00:00] [DBG] [20] MediaBrowser.Providers.Manager.ProviderManager: OnRefreshStart "61c590960cac514e5f6c313f8b3d3266"
[2022-11-22 15:57:11.159 +00:00] [DBG] [8] MediaBrowser.Providers.Manager.ProviderManager: OnRefreshStart "1071671e7bffa0532e930debee501d2e"
[2022-11-22 15:57:11.170 +00:00] [DBG] [8] MediaBrowser.Providers.Manager.ProviderManager: OnRefreshProgress "f27caa37e5142225cceded48f6553502" 11
[2022-11-22 15:57:11.170 +00:00] [DBG] [8] MediaBrowser.Providers.Manager.ProviderManager: OnRefreshProgress "1071671e7bffa0532e930debee501d2e" 5
[2022-11-22 15:57:11.170 +00:00] [DBG] [8] MediaBrowser.Providers.Manager.ProviderManager: OnRefreshProgress "f27caa37e5142225cceded48f6553502" 12
[2022-11-22 15:57:11.170 +00:00] [DBG] [8] MediaBrowser.Providers.Manager.ProviderManager: OnRefreshProgress "1071671e7bffa0532e930debee501d2e" 10
[2022-11-22 15:57:11.171 +00:00] [DBG] [21] MediaBrowser.Providers.Manager.ProviderManager: OnRefreshComplete "1071671e7bffa0532e930debee501d2e"
[2022-11-22 15:57:11.175 +00:00] [DBG] [21] MediaBrowser.Providers.Manager.ProviderManager: OnRefreshProgress "f27caa37e5142225cceded48f6553502" 30
[2022-11-22 15:57:11.214 +00:00] [DBG] [20] MediaBrowser.Providers.Manager.ProviderManager: OnRefreshProgress "f27caa37e5142225cceded48f6553502" 31
[2022-11-22 15:57:11.214 +00:00] [DBG] [20] MediaBrowser.Providers.Manager.ProviderManager: OnRefreshProgress "61c590960cac514e5f6c313f8b3d3266" 5
[2022-11-22 15:57:11.217 +00:00] [DBG] [20] MediaBrowser.Providers.Manager.ProviderManager: OnRefreshProgress "f27caa37e5142225cceded48f6553502" 32
[2022-11-22 15:57:11.217 +00:00] [DBG] [20] MediaBrowser.Providers.Manager.ProviderManager: OnRefreshProgress "61c590960cac514e5f6c313f8b3d3266" 10
[2022-11-22 15:57:11.218 +00:00] [DBG] [20] MediaBrowser.Providers.Manager.ProviderManager: OnRefreshComplete "61c590960cac514e5f6c313f8b3d3266"
[2022-11-22 15:57:11.218 +00:00] [DBG] [20] MediaBrowser.Providers.Manager.ProviderManager: OnRefreshProgress "f27caa37e5142225cceded48f6553502" 50
[2022-11-22 15:57:11.218 +00:00] [DBG] [20] MediaBrowser.Providers.Manager.ProviderManager: OnRefreshProgress "f27caa37e5142225cceded48f6553502" 50
[2022-11-22 15:57:11.220 +00:00] [DBG] [10] MediaBrowser.Providers.Manager.ProviderManager: OnRefreshProgress "f27caa37e5142225cceded48f6553502" 75
[2022-11-22 15:57:11.220 +00:00] [DBG] [21] MediaBrowser.Providers.Manager.ProviderManager: OnRefreshProgress "f27caa37e5142225cceded48f6553502" 100
[2022-11-22 15:57:11.220 +00:00] [DBG] [23] MediaBrowser.Providers.Manager.ProviderManager: OnRefreshComplete "f27caa37e5142225cceded48f6553502"
[2022-11-22 15:57:11.223 +00:00] [DBG] [23] Emby.Server.Implementations.Library.LibraryManager: Running post-scan task "SplashscreenPostScanTask"
[2022-11-22 15:57:11.224 +00:00] [DBG] [23] Emby.Server.Implementations.Library.SplashscreenPostScanTask: No thumb images found. Using backdrops to generate splashscreen
[2022-11-22 15:57:11.244 +00:00] [ERR] [23] Emby.Server.Implementations.Library.LibraryManager: Error running post-scan task
System.ArgumentException: Not enough valid pictures provided to create a splashscreen!
   at Jellyfin.Drawing.Skia.SplashscreenBuilder.GenerateCollage(IReadOnlyList`1 posters, IReadOnlyList`1 backdrops)
   at Jellyfin.Drawing.Skia.SplashscreenBuilder.GenerateSplash(IReadOnlyList`1 posters, IReadOnlyList`1 backdrops, String outputPath)
   at Jellyfin.Drawing.Skia.SkiaEncoder.CreateSplashscreen(IReadOnlyList`1 posters, IReadOnlyList`1 backdrops)
   at Emby.Server.Implementations.Library.SplashscreenPostScanTask.Run(IProgress`1 progress, CancellationToken cancellationToken)
   at Emby.Server.Implementations.Library.LibraryManager.RunPostScanTasks(IProgress`1 progress, CancellationToken cancellationToken)
[2022-11-22 15:57:11.247 +00:00] [DBG] [23] Emby.Server.Implementations.Library.LibraryManager: Running post-scan task "ArtistsPostScanTask"
[2022-11-22 15:57:11.253 +00:00] [DBG] [23] Emby.Server.Implementations.Library.LibraryManager: Running post-scan task "CollectionPostScanTask"
[2022-11-22 15:57:11.258 +00:00] [DBG] [23] Emby.Server.Implementations.Library.LibraryManager: Running post-scan task "GenresPostScanTask"
[2022-11-22 15:57:11.260 +00:00] [DBG] [23] Emby.Server.Implementations.Library.LibraryManager: Running post-scan task "MusicGenresPostScanTask"
[2022-11-22 15:57:11.263 +00:00] [DBG] [23] Emby.Server.Implementations.Library.LibraryManager: Running post-scan task "StudiosPostScanTask"
[2022-11-22 15:57:11.266 +00:00] [DBG] [23] Emby.Server.Implementations.Library.LibraryManager: Running post-scan task "CleanDatabaseScheduledTask"
[2022-11-22 15:57:11.269 +00:00] [DBG] [23] Emby.Server.Implementations.Data.CleanDatabaseScheduledTask: Cleaning 0 items with dead parent links
[2022-11-22 15:57:11.270 +00:00] [INF] [23] Emby.Server.Implementations.ScheduledTasks.TaskManager: "Scan Media Library" Completed after 0 minute(s) and 0 seconds
[2022-11-22 15:57:11.271 +00:00] [INF] [22] Emby.Server.Implementations.IO.LibraryMonitor: Watching directory "/media/samples"
[2022-11-22 15:57:11.271 +00:00] [INF] [23] Emby.Server.Implementations.ScheduledTasks.TaskManager: ExecuteQueuedTasks
[2022-11-22 15:57:12.685 +00:00] [DBG] [26] Emby.Server.Implementations.HttpServer.WebSocketConnection: WS "*.*.*.*" received message: WebSocketMessage`1 { MessageType: KeepAlive, MessageId: 00000000-0000-0000-0000-000000000000, ServerId: null, Data: null }
[2022-11-22 15:57:13.624 +00:00] [DBG] [23] Emby.Server.Implementations.HttpServer.WebSocketConnection: WS "*.*.*.*" received message: WebSocketMessage`1 { MessageType: ActivityLogEntryStop, MessageId: 00000000-0000-0000-0000-000000000000, ServerId: null, Data: JsonElement { ValueKind: String } }
[2022-11-22 15:57:13.624 +00:00] [DBG] [23] Jellyfin.Api.WebSocketListeners.ActivityLogWebSocketListener: WS "*.*.*.*" stop transmitting to "ActivityLogWebSocketListener"
[2022-11-22 15:57:13.625 +00:00] [DBG] [23] Emby.Server.Implementations.HttpServer.WebSocketConnection: WS "*.*.*.*" received message: WebSocketMessage`1 { MessageType: ActivityLogEntryStop, MessageId: 00000000-0000-0000-0000-000000000000, ServerId: null, Data: JsonElement { ValueKind: String } }
[2022-11-22 15:57:13.625 +00:00] [DBG] [23] Jellyfin.Api.WebSocketListeners.ActivityLogWebSocketListener: WS "*.*.*.*" stop transmitting to "ActivityLogWebSocketListener"
[2022-11-22 15:57:13.625 +00:00] [DBG] [23] Emby.Server.Implementations.HttpServer.WebSocketConnection: WS "*.*.*.*" received message: WebSocketMessage`1 { MessageType: ScheduledTasksInfoStop, MessageId: 00000000-0000-0000-0000-000000000000, ServerId: null, Data: null }
[2022-11-22 15:57:13.625 +00:00] [DBG] [23] Jellyfin.Api.WebSocketListeners.ScheduledTasksWebSocketListener: WS "*.*.*.*" stop transmitting to "ScheduledTasksWebSocketListener"
[2022-11-22 15:57:13.669 +00:00] [DBG] [10] Jellyfin.Api.Auth.CustomAuthenticationHandler: AuthenticationScheme: "CustomAuthentication" was successfully authenticated.
[2022-11-22 15:57:15.078 +00:00] [DBG] [23] Jellyfin.Api.ModelBinders.CommaDelimitedArrayModelBinder: Error converting value.
System.FormatException: SongCount is not a valid value for ItemFields.
 ---> System.ArgumentException: Requested value 'SongCount' was not found.
   at System.Enum.TryParseByName(RuntimeType enumType, ReadOnlySpan`1 value, Boolean ignoreCase, Boolean throwOnFailure, UInt64& result)
   at System.Enum.TryParse(Type enumType, ReadOnlySpan`1 value, Boolean ignoreCase, Boolean throwOnFailure, Object& result)
   at System.Enum.TryParse(Type enumType, String value, Boolean ignoreCase, Boolean throwOnFailure, Object& result)
   at System.ComponentModel.EnumConverter.ConvertFrom(ITypeDescriptorContext context, CultureInfo culture, Object value)
   --- End of inner exception stack trace ---
   at System.ComponentModel.EnumConverter.ConvertFrom(ITypeDescriptorContext context, CultureInfo culture, Object value)
   at Jellyfin.Api.ModelBinders.CommaDelimitedArrayModelBinder.GetParsedResult(IReadOnlyList`1 values, Type elementType, TypeConverter converter)
[2022-11-22 15:57:17.922 +00:00] [DBG] [19] Jellyfin.Api.Auth.CustomAuthenticationHandler: AuthenticationScheme: "CustomAuthentication" was successfully authenticated.
```

**Comment by cassiofb-dev** [comment]
> I had a clean install and had the same issue, as a new comer to jellyfish its pretty sad. I've seen a couple of mentions of this issue but no clear way of fixing. I'm going to reinstall and then disable splash screen and see what happens

@VincentHui I am having the same issue, did you solve it?

**Comment by mrVanDalo** [comment]
I had this issue, and no it turned out, I had a `.ignore` file the toplevel of the folder I would wanted to put in the library. This told jellyfin to skip this folder. `rm .ignore` fixed my issues.

**Comment by veevee2** [comment]
hi 
i had the same issue, and it seems cache or metadatas were corrupted.
Solution for me : 

1. I stopped, jellyfin
2. deleted the cache and metadata folders
3. restarted jellyfin
4. launch a medias refresh
those folder are setted in general settings :
![image](https://user-images.githubusercontent.com/62995697/226205114-688323be-bb4d-44a7-ad77-9f546cb102ee.png)

**Comment by veevee2** [comment]
if it worked for you, that should significate a fix is required to handle that situation with no user technical action needed.

**Comment by jeames00** [comment]
This issue persists for me with the latest version of Jellyfin Server (10.8.9). I emptied the library and config directories, pulled the new docker image (lscr.io/linuxserver/jellyfin:latest), then brought up the docker container. The same error about the splash screen is printed in the logs.

> [2023-03-20 13:50:38.983 +11:00] [INF] [22] Emby.Server.Implementations.ScheduledTasks.TaskManager: Executing "Scan Media Library"
[2023-03-20 13:50:38.993 +11:00] [INF] [22] Emby.Server.Implementations.Library.LibraryManager: Validating media library
[2023-03-20 13:50:40.958 +11:00] [ERR] [22] Emby.Server.Implementations.Library.LibraryManager: Error running post-scan task
System.ArgumentException: Not enough valid pictures provided to create a splashscreen!
   at Jellyfin.Drawing.Skia.SplashscreenBuilder.GenerateCollage(IReadOnlyList\`1 posters, IReadOnlyList\`1 backdrops)
   at Jellyfin.Drawing.Skia.SplashscreenBuilder.GenerateSplash(IReadOnlyList\`1 posters, IReadOnlyList\`1 backdrops, String outputPath)
   at Jellyfin.Drawing.Skia.SkiaEncoder.CreateSplashscreen(IReadOnlyList\`1 posters, IReadOnlyList\`1 backdrops)
   at Emby.Server.Implementations.Library.SplashscreenPostScanTask.Run(IProgress\`1 progress, CancellationToken cancellationToken)
   at Emby.Server.Implementations.Library.LibraryManager.RunPostScanTasks(IProgress\`1 progress, CancellationToken cancellationToken)
[2023-03-20 13:50:41.051 +11:00] [INF] [22] Emby.Server.Implementations.ScheduledTasks.TaskManager: "Scan Media Library" Completed after 0 minute(s) and 2 seconds

Could this be a permissions problem with my library folders on the NFS share? They currently are set to "rwxr-xr-x" for user "dockrema" and group "nogroup".

**Comment by JueLuo99** [comment]
I have the same issue.

**Comment by nonprenom** [comment]
I have the same issue.

Version: 10.8.10
Operating System: Linux (Debian)
Architecture: X64
(it's not docker install)

logs of scanning part :

```
[2023-06-06 09:27:11.753 +03:00] [INF] Executing "Scan Media Library"
[2023-06-06 09:27:11.764 +03:00] [INF] Stopping directory watching for path "/opt/Videos/Movies"
[2023-06-06 09:27:11.765 +03:00] [INF] Stopping directory watching for path "/opt/Videos/TVShows"
[2023-06-06 09:27:11.767 +03:00] [INF] Validating media library
[2023-06-06 09:28:52.151 +03:00] [ERR] Error running post-scan task
System.ArgumentException: Not enough valid pictures provided to create a splashscreen!
   at Jellyfin.Drawing.Skia.SplashscreenBuilder.GenerateCollage(IReadOnlyList`1 posters, IReadOnlyList`1 backdrops)
   at Jellyfin.Drawing.Skia.SplashscreenBuilder.GenerateSplash(IReadOnlyList`1 posters, IReadOnlyList`1 backdrops, String outputPath)
   at Jellyfin.Drawing.Skia.SkiaEncoder.CreateSplashscreen(IReadOnlyList`1 posters, IReadOnlyList`1 backdrops)
   at Emby.Server.Implementations.Library.SplashscreenPostScanTask.Run(IProgress`1 progress, CancellationToken cancellationToken)
   at Emby.Server.Implementations.Library.LibraryManager.RunPostScanTasks(IProgress`1 progress, CancellationToken cancellationToken)
[2023-06-06 09:28:52.171 +03:00] [INF] "Scan Media Library" Completed after 1 minute(s) and 40 seconds

```

**Comment by Gloryandel** [comment]
Just switched from Windows Server to Linux Docker and can't perform a scan.
```
[2023-06-12 17:42:22.091 +08:00] [ERR] [17] Emby.Server.Implementations.Library.LibraryManager: Error running post-scan task
System.ArgumentException: Not enough valid pictures provided to create a splashscreen!
   at Jellyfin.Drawing.Skia.SplashscreenBuilder.GenerateCollage(IReadOnlyList`1 posters, IReadOnlyList`1 backdrops)
   at Jellyfin.Drawing.Skia.SplashscreenBuilder.GenerateSplash(IReadOnlyList`1 posters, IReadOnlyList`1 backdrops, String outputPath)
   at Jellyfin.Drawing.Skia.SkiaEncoder.CreateSplashscreen(IReadOnlyList`1 posters, IReadOnlyList`1 backdrops)
   at Emby.Server.Implementations.Library.SplashscreenPostScanTask.Run(IProgress`1 progress, CancellationToken cancellationToken)
   at Emby.Server.Implementations.Library.LibraryManager.RunPostScanTasks(IProgress`1 progress, CancellationToken cancellationToken)
[2023-06-12 17:42:22.124 +08:00] [INF] [17] Emby.Server.Implementations.ScheduledTasks.TaskManager: "扫描媒体库" Completed after 1 minute(s) and 40 seconds
[2023-06-12 17:42:22.126 +08:00] [INF] [17] Emby.Server.Implementations.ScheduledTasks.TaskManager: Execute Queued Tasks
[2023-06-12 17:42:22.150 +08:00] [INF] [21] Emby.Server.Implementations.IO.LibraryMonitor: Watching directory "/videos/测试"
```

**Comment by nielsvanvelzen** [comment]
The "Not enough valid pictures provided to create a splashscreen!" message can be safely ignores. It's a post-scan task and doesn't impact the scanning at all.

**Comment by Gloryandel** [comment]
This is a comparison between Jellyfin and Emby, the files and directories are the same and Jellyfin cannot complete the scan. The previous Windows Jellyfin worked fine. The metadata is all there. Have you encountered this situation?
- Emby
![Snipaste_2023-06-12_19-00-56](https://github.com/jellyfin/jellyfin/assets/51497852/b7e9bac1-3d97-4e0a-a6ff-fee25e7d52b7)  
![Snipaste_2023-06-12_19-06-52](https://github.com/jellyfin/jellyfin/assets/51497852/d1b4553b-bf25-42df-bf69-16f3c3893dbb)  

- Jellyfin
![Snipaste_2023-06-12_19-09-49](https://github.com/jellyfin/jellyfin/assets/51497852/1fd7178a-944c-42dc-9081-37b86c982a28)  
![Snipaste_2023-06-12_19-15-41](https://github.com/jellyfin/jellyfin/assets/51497852/585b2426-21d7-40c8-8e0b-138c213e67cd)

**Comment by galeksandrp** [comment]
Seems like Jellyfin having permission issue for library directory path.

For example, on Windows, I encountered same behavior from the beginning. I needed to manually add at least READ permission to `NETWORK SERVICE` in my `libraries` folders.

<details>
<summary>How to add permission to folder</summary>

- Right clicka on folder to open context menu
- In context menu click `Properties` (on Windows 11 click `Additional parameters`, then `Properties`)
- In opened window go to `Security` tab
- Click `Edit` button
- Click `Add............` button
- Paste `NETWORK SERVICE` (or click `Advanced`, then `Find now`, and select `NETWORK SERVICE` from the list).
- Click `OK` thousand times until all windows close
- Rescan libraries in Jellyfin.

![image](https://github.com/jellyfin/jellyfin/assets/3586205/e3759187-1830-46f8-b627-0e11ae17543c)

</details>
